cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/schedule/store/DatasetBasedTimeScheduleStoreTest.java

```diff
@@ -359,19 +352,7 @@ public void testTriggersWithoutJobs() throws Exception {
     verifyJobAndTriggers(job2.getKey(), 1, Trigger.TriggerState.NORMAL);
 
     // Delete the row corresponding to the Job MR1 to create inconsistency
-    final Table table = tableUtil.getMetaTable();
-
-    factory.createExecutor(ImmutableList.of((TransactionAware) table))
-      .execute(new TransactionExecutor.Subroutine() {
-        @Override
-        public void apply() throws Exception {
-          datasetBasedTimeScheduleStore.removeJob(table, job1.getKey());
-        }
-      });
-
-    // Make sure Jobs and Triggers are setup correctly
-    verifyJobAndTriggers(job1.getKey(), 1, Trigger.TriggerState.NORMAL);
```

Comment:
why is this removed?

Reply:
It's done above. With the new implementation, it doesn't make sense to check after I remove job1 - it'll already be gone by then